### DISCUSSION TITLE
Scan Nvidia API if the Product_url has changed - but how?

### DISCUSSION THREAD

**Discussion by rasurbrand** [jef/streetmerchant#3051] [Help]
Hi 
my idea is to check the Nvidia Api: https://api.store.nvidia.com/partner/v1/feinventory?status=1&skus=NVGFT490&locale=DE

I would like to check if the **product_url** has changed.
I tried the following code.
Idea: if the container "**.url**" still has the old url "https://www.notebooksbilliger.de/nvidia+geforce+rtx+4090+founders+edition+782690" the url hasnt changed and the status is "**outofStock**" but if the value is something different, that value has changed and the item is "**inStock**" and I get a notification.

`

> import {Store} from './store';
> 
> export const NvidiaDE: Store = {
>   currency: '€',
>   labels: {
>     outOfStock: {
>       container: '.url',
>       text: ['https://www.notebooksbilliger.de/nvidia+geforce+rtx+4090+founders+edition+782690'],
>     },
>   },
>   links: [
>     {
>       brand: 'nvidia',
>       model: 'founders edition',
>       series: '4090',
>       url: 'https://api.store.nvidia.com/partner/v1/feinventory?status=1&skus=NVGFT490&locale=DE',
>     },
>    
>   ],
>   name: 'nvidia-de',
> };

`

How ever, the problem is, that the container ".url" seems to have no "text" in it but a "title" for example.

`<a class="url" title="https://www.notebooksbilliger.de/nvidia+geforce+rtx+4090+founders+edition+782690" draggable="false" href="https://www.notebooksbilliger.de/nvidia+geforce+rtx+4090+founders+edition+782690" target="_blank" rel="noopener noreferrer" tabindex="-1">https://www.notebooksbilliger.de/nvidia+geforce+rtx+4090+founders+edition+782690</a>`

So I would like to check the "title" or "href" information if this is possible? Maybe someone has an idea
But maybe my fault is somewhere else? wrong container? Any help is appreciated :)

**Comment by rasurbrand** [comment]
up

**Reply by Mr-Tech-13** [reply]
Im not sure how to say this, but you need to make href a serchable object. If that makes no sense, ill find a better way to word it.
